### PR TITLE
adjust print statement to understand error on jenkins only

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Got an error on jenkins run that I can't reproduce locally. This tiny change might let me understand it better.

(The suggestion on https://jenkins.aws.ahconu.org/view/Docstore/job/docstore-dev-drush/88/console to use --no-halt-on-error wasn't useful: https://jenkins.aws.ahconu.org/view/Docstore/job/docstore-dev-drush/89/console)

### CODE REVIEW

**Comment by lazysoundsystem** [comment]
Not sure what to make of that - I got confused by jenkins complaining about the lack of quotes (which wasn't a problem when I ran the command before) https://jenkins.aws.ahconu.org/view/Docstore/job/docstore-dev-drush/90/console - ERROR: script returned exit code 129

When I tried again, with quotes, the problematic `object` of https://jenkins.aws.ahconu.org/view/Docstore/job/docstore-dev-drush/88/console turned out to be a string after all:
https://jenkins.aws.ahconu.org/view/Docstore/job/docstore-dev-drush/92/console

In short, the script did (finally) run as expected on staging.
